source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md

```diff
@@ -79,3 +79,9 @@ Training sessions during the event were completely packed, and some people found
 by Ben Weiss
 
 During his talk, Ben showed how to improve our app discoverability using Instant Apps where the key is modularising our apps. The primary constraint here seems to be the size of our modules which should be smaller than four megabytes. Also recommended tools to deal with this are the apk analyzer and configuration splits on top of the usual minification, shrinking and proguard obfuscation.
+
+# Final thoughts
+
+Droidcon Berlin was great to find out more about a wide range of topics related to Android and meet interesting people. GDD Krakow was a very big event where I have the opportunity to help in a training session, meet some of my mentees at Udacity and discuss different topics with Google Developer Advocates. 
+
+Taking three flights for two conferences in a row in different countries is probably as tiring as it seems. Nevertheless, it was an awesome experience and I could not be happier about it.
```

Comment:
Maybe touch upon the talks as well? Best talk, most surprisingly interesting - that kind